### PR TITLE
master and worker containers should start after reboot

### PR DESCRIPTION
**Pull request by batikanu**
This should solve reboot problem

### CODE REVIEW

**Comment by batikanu** [comment]
@zreigz @taimir @cheld what do you think?

**Comment by batikanu** [comment]
It works for only node now.

**Comment by taimir** [comment]
We have 3 different restart policies:
- `on-failure`, with some limit on retries
- `always`
- `unless-stopped`

Which want should we pick here? I personally tend towards `on-failure` with some buffer for possible retries.

**Comment by batikanu** [comment]
For master node is not yet functional.

**Comment by zreigz** [comment]
It partially solve the problem, only for minions. Currently I am testing my solution to fully fix the issue.

File: fragments/master.sh [hunk 1]
```diff
@@ -198,6 +198,7 @@ start_k8s(){
         --net=host \
         --pid=host \
         --privileged=true \
+        --restart=on-failure \
```
Comment (zreigz) [inline fragments/master.sh]: Please remove this because it doesn't fix the problem.

**Comment by zreigz** [comment]
You can leave this policy only for worker.

**Comment by zreigz** [comment]
PTAL

**Comment by batikanu** [comment]
Yes for master not functional yet. PTAL

**Comment by zreigz** [comment]
LGTM

**Pull request merged**